changelog/fragments/1677797191-dispatch-periodic-flush.yaml

```diff
@@ -0,0 +1,34 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: bug-fix
+
+# Change summary; a 80ish characters long description of the change.
+summary: Periodically run the dispatcher from a timer.
```

Comment:
Can you rephrase this to describe the impact to the user? What bug does this fix? What features are effected?

The descriptions here are good but they will only really make sense to engineers, the change log is for users.

Reply:
None of our users know what the dispatcher is or does.

Reply:
That is an excellent changelog entry, thanks!